cluster/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -xe
+
+make cluster-down
```

Comment:
Most of these targets don't currently exist. Also, I may be able to be convinced otherwise but I think humans and travis ci are the only things off-hand that should be calling `make`, scripts should include/execute other scripts instead.

Reply:
This PR depends on PR#87, the targets exist there.